content/configuration.js

```diff
@@ -30,13 +34,13 @@ module.exports = {
 
     // menus
     menus: menus,
-  },
 
-  languages: {
-    default: 'en',
-    allowed: ['en', 'en-US', 'en-GB', 'fr', 'none'],
+    //
+    languages: languages,
   },
 
+  languages: languages,
```

Comment:
@jonhartzler  I'd like to move this to the siteMetadata section so it's a bit easier to pass around within the site, but I didn't want to touch your scripts within the plugins folder since you're actively working there. So for now this is duplicated.